bibangamba/GitHubUsers
# #155965831 Display List of Github Users

#### What does this PR do?
Display List of Github Users

#### Description of tasks to be done?
- connect the GithubUsersPresenter to the MainActivity
- use the interface (GithubUsersView) to ensure the recyclerview is populated when we get data

#### How can this be manually tested?
clone the branch and run the application. It should start out on the list of GitHub users

#### What are the relevant pivotal tracker stories?
[#155965831](https://www.pivotaltracker.com/story/show/155965831)

#### Screenshots
![image](https://user-images.githubusercontent.com/5712103/53641944-7e3d5400-3c41-11e9-9f1e-82fb19658fcf.png)
